ui/tests/acceptance/enterprise-kmip-test.js

```diff
@@ -22,14 +22,15 @@ import rolesPage from 'vault/tests/pages/secrets/backend/kmip/roles';
 import credentialsPage from 'vault/tests/pages/secrets/backend/kmip/credentials';
 import mountSecrets from 'vault/tests/pages/settings/mount-secret-backend';
 import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { mountBackend } from 'vault/tests/helpers/components/mount-backend-form-helpers';
 import { allEngines } from 'vault/helpers/mountable-secret-engines';
 import { mountEngineCmd, runCmd } from 'vault/tests/helpers/commands';
 import { v4 as uuidv4 } from 'uuid';
 
 // port has a lower limit of 1024
 const getRandomPort = () => Math.floor(Math.random() * 5000 + 1024);
 
-const mount = async (backend) => {
+const mountWithCli = async (backend) => {
```

Comment:
We could remove this helper and replace with `mountEngineCmd(backend, 'kmip')` below for consistency